src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -63,7 +65,13 @@ private void setup(boolean addShutdownHook, Settings settings, Environment envir
             Natives.tryMlockall();
         }
 
-        NodeBuilder nodeBuilder = NodeBuilder.nodeBuilder().settings(settings).loadConfigSettings(false);
+        // We do not need to reload system properties here as we have already applied them in building the settings
+        Settings nodeSettings = ImmutableSettings.settingsBuilder()
+                .put(settings)
+                .put("config.ignore_system_properties", true)
```

Comment:
can we document why we do it (and make this a constant since we use it in several places now)?